<launch>
	<!-- The mp3 recorder -->
   <!-- <node name="recorder" type="recorder" pkg="utilite">
    	
    	/media/Macintosh HD/Users/MatLab/Music/2012
    	/home/mathieu/inf_audio_saved.mp3
    	
    	<param name="path" value="" type="string"/>
    </node>-->
    
    <!-- The overlappers -->
    <node name="overlap10" type="overlap" pkg="utilite">
    	<param name="ratio" value="0.10" type="double"/>
    	<remap from="audioOverlappedFrameFreqSqrdMagn" to="freqOverlap10" />
    </node>
    <node name="overlap25" type="overlap" pkg="utilite">
    	<param name="ratio" value="0.25" type="double"/>
    	<remap from="audioOverlappedFrameFreqSqrdMagn" to="freqOverlap25" />
    </node>
    <node name="overlap50" type="overlap" pkg="utilite">
    	<param name="ratio" value="0.5" type="double"/>
    	<remap from="audioOverlappedFrameFreqSqrdMagn" to="freqOverlap50" />
    </node>
    <node name="overlap100" type="overlap" pkg="utilite">
    	<param name="ratio" value="1" type="double"/>
    	<remap from="audioOverlappedFrameFreqSqrdMagn" to="freqOverlap100" />
    </node>
    
    <!-- The players (only the first play the music) -->
    <node name="player0" type="player" pkg="utilite">
    	<param name="gui_used" value="true" type="bool"/>
    </node>
    <node name="player10" type="player" pkg="utilite">
    	<remap from="audioFrame" to="dummy" />
    	<remap from="audioFrameFreqSqrdMagn" to="freqOverlap10" />
    </node>
    <node name="player25" type="player" pkg="utilite">
    	<remap from="audioFrame" to="dummy" />
    	<remap from="audioFrameFreqSqrdMagn" to="freqOverlap25" />
    </node>
    <node name="player50" type="player" pkg="utilite">
    	<remap from="audioFrame" to="dummy" />
    	<remap from="audioFrameFreqSqrdMagn" to="freqOverlap50" />
    </node>
    <node name="player100" type="player" pkg="utilite">
    	<remap from="audioFrame" to="dummy" />
    	<remap from="audioFrameFreqSqrdMagn" to="freqOverlap100" />
    </node>
    
</launch>
